var/spack/repos/builtin/packages/py-mda-xdrlib/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMdaXdrlib(PythonPackage):
+    """A stand-alone XDRLIB module extracted from CPython 3.10.8"""
+
+    homepage = "https://github.com/MDAnalysis/mda-xdrlib"
+    pypi = "mda_xdrlib/mda_xdrlib-0.2.0.tar.gz"
+
+    maintainers("RMeli")
+
+    version("0.2.0", sha256="f26f7158a83c32b96d15b530fce2cbc1190c4b7024e41faa4ab3e3db74e272af")
+
+    depends_on("python@3.7:3.11", type=("build", "run"))
+    depends_on("py-wheel", type="build")
```

Comment:
`py-wheel` is added as dependency by default. You would only need to specify it again if a certain version is needed.